config/NEI/collapsibleitems.cfg

```diff
@@ -223,6 +241,14 @@ $pipeHexadecuple
 :gt.metatool.01 180-192
 #gregtech coffe
 :gt.metaitem.02 32000-32009
+#Exquisite
+:gt.metaitem.02 30208,30209,30500-30505,30507-30514,30527,30528,30890 | dreamcraft:item. GemExquisite
```

Comment:
wouldnt it be more helpful to group by material instead of by oreprefix?

Reply:
I don't think so. Groups are more useful if used to group by type rather than by material.

Reply:
P.S. We group the same Storage Drawers by type and not by the material they are made of

Reply:
because they are interchangeable and practically the same item. nothing could be further from the truth for different dusts/plates/etc

Reply:
Gems are no different from plates, dust, gears, wires, pipes, etc.

Reply:
When I started creating groups I tried to follow simple grouping rules:

1. objects of the same type but different colors (example: glass)
2. items of the same tier but from different materials (example: drawers,gears,wires)
3. association by topic. not advisable. rarely used (example: flower)

Reply:
If I need Tungstensteel Double Plates, I would like to search for "Tungstensteel" and expand the "Tungstensteel" group instead of having to search for "Double Plates", expand the "Double Plate" group and *additonally* search for "Tungstensteel Double Plate" in that group.

Reply:
this is what you get if you search "Tungstensteel"
![image](https://github.com/user-attachments/assets/50734e0e-6606-471e-90dc-50db73c90f18)

You won't have to expand anything. But if you make groups by material, you'll have to expand the material every time